src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/PileupSummary.java

```diff
@@ -107,6 +141,31 @@ public static void writeToFile(final String sample, final List<PileupSummary> re
         }
     }
 
+    public static class PileupSummaryComparator implements Comparator<PileupSummary> {
+        final SAMSequenceDictionary sequenceDictionary;
+        final List<String> contigsInOrder;
+
+        public PileupSummaryComparator(final SAMSequenceDictionary sequenceDictionary){
+            this.sequenceDictionary = sequenceDictionary;
+            contigsInOrder = sequenceDictionary.getSequences().stream()
+                    .map(ssr -> ssr.getSequenceName())
```

Comment:
This looks like alphabetical order; is that right?  If so, would the sequence dictionary's own order make more sense?

Reply:
I'm using the order in the sequence dictionary, unless I'm missing implicit sorting that takes places somewhere